### ISSUE TITLE
training problems

### ISSUE THREAD

**Issue by melodyhappy** [carla-simulator/imitation-learning#42]
i use your source training code  to training from scratch, but i found when the loss decreased to 2, it start to fluctuate between 3 and 7.so it that reflects that the training is overfitting?
and are there any training tricks? Hoping for your answer.THX

**Comment by margonotmango** [comment]
hi,
Could you plz tell me where the source training code is? I can't find it. thanks

**Comment by pri2si17-1997** [comment]
@melodyhappy can you please tell where source code of training model is located?

**Comment by melodyhappy** [comment]
@margonotmango @pri2si17-1997  the source training code is here https://github.com/mvpcom/carlaILTrainer

**Comment by markus-hinsche** [comment]
Here is another code that also does the training, which might or might not be easier to read :)
https://github.com/merantix/imitation-learning
